app/src/main/java/com/m2049r/xmrwallet/util/Helper.java

```diff
@@ -240,4 +242,17 @@ static public void clipBoardCopy(Context context, String label, String text) {
         ClipData clip = ClipData.newPlainText(label, text);
         clipboardManager.setPrimaryClip(clip);
     }
+
+    static private OkHttpClient OkHttpClientSingleton;
```

Comment:
Maybe put this into a dedicated class, single responsibility.

Reply:
yes, done.